pkg/querier/engine/streaming/operator/time.go

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package operator
+
+import "time"
+
+func stepCount(start, end, interval int64) int {
+	return int((end-start)/interval) + 1
+}
+
+func DurationMilliseconds(d time.Duration) int64 {
+	return int64(d / (time.Millisecond / time.Nanosecond))
```

Comment:
Can't you use `d.Milliseconds()` here?

Reply:
Good catch... I took this from Prometheus' code, not quite sure why they're doing this, but `d.Milliseconds()` does the exact same thing.

Fixed in ba06a32.